com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -557,6 +553,10 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
                 }
             }
 
+            using (new ProfilingScope(null, Profiling.Pipeline.endCameraRendering))
+            {
+                EndCameraRendering(context, baseCamera);
+            }
```

Comment:
This doesn't look correct. EndCameraRendering for overlay cameras is called before the EndCameraRendering for base camera.

Reply:
Addressed in f1a61a9